go/v1beta1/storage/elasticsearch.go

```diff
@@ -159,9 +169,17 @@ func (es *ElasticsearchStorage) GetProject(ctx context.Context, projectID string
 	projectName := fmt.Sprintf("projects/%s", projectID)
 	log := es.logger.Named("GetProject").With(zap.String("project", projectName))
 
-	searchTerm, err := createElasticsearchSearchTermQuery(map[string]interface{}{
-		"name": projectName,
-	})
+	filterQuery, errs := filtering.ParseExpressionEntrypoint(fmt.Sprintf(`name=="%s"`, projectName))
```

Comment:
We don't really need to do this if the filter is already known beforehand, right?  It's always going to be the same search query.  We should update this to search in the same way that `CreateProject` does.